modules/ebi-metagenomics/foldcomp/extract/main.nf

```diff
@@ -0,0 +1,51 @@
+process FOLDCOMP_EXTRACT {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/foldcomp:0.0.5--h43eeafb_2':
+        'biocontainers/foldcomp:0.0.5--h43eeafb_2' }"
+
+    input:
+    tuple val(meta), path(fcz)
+    val(mode)
+
+    output:
+    tuple val(meta), path("{${fcz}_plddt,*.plddt}"), optional: true, emit: plddt
+    tuple val(meta), path("{${fcz}_fasta,*.fasta}"), optional: true, emit: fasta
+    path "versions.yml"                            , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def mode_arg = mode == "fasta" ? "--fasta" : "--plddt"
```

Comment:
Hmm, I haven't seen other modules in nf-core going that far. I think that is because nf-modules are not being directly used by your average user. The end user will most often install and run a pipeline and that's where all the checks (nf-validation etc) are mostly happening. Modules are mostly used by other developers and are kept in a minimal version.

Reply:
that makes sense to me!! definitely too nitpicky then so ignore 😂